spec/features/sessions_spec.rb

```diff
@@ -1,28 +1,28 @@
-require "rails_helper"
+# require "rails_helper"
 
 
-feature "A user tries to log in" do
+# # feature "A user tries to log in" do
 
-  scenario "a user tries to log in with username and see their personal user page" do
-    user = create(:user)
-    visit login_path
-    fill_in "session_user_name", with: user.username
-    fill_in "session_user_password", with: user.password
-    click_button "Log In"
+# #   scenario "a user tries to log in with username and see their personal user page" do
+# #     user = create(:user)
+# #     visit login_path
+# #     fill_in "session_user_name", with: user.username
+# #     fill_in "session_user_password", with: user.password
+# #     click_button "Log In"
 
-    visit profile_path
-    expect(page).to have_content user.username
-  end
+# #     visit profile_path
+# #     expect(page).to have_content user.username
+# #   end
 
-  scenario "a user tries to log in with email and see their personal user page" do
-    user = create(:user)
-    visit login_path
-    fill_in "session_user_name", with: user.email
-    fill_in "session_user_password", with: user.password
-    click_button "Log In"
+# #   scenario "a user tries to log in with email and see their personal user page" do
```

Comment:
![tumblr_mgkokoikad1s0xrbio1_250](https://cloud.githubusercontent.com/assets/50019/5658719/d18d6670-96d4-11e4-9b19-f1b669229f0c.gif)

It's doubly commented out so it's double invisible.